docs/core-concepts/accounts.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Accounts
+---
+
+# Accounts
+
+Accounts within Solana are used to store state. They are an essential
+building block for developing on Solana.
+
+## Facts
+
+::: tip Fact Sheet
+- Accounts are used to store data
+- Each account has a unique address
+- Accounts have a max size of 10MB
+- Account size is static
+- Account data storage is paid with rent
+- Default account owner is the System Program
+:::
+
+## Deep Dive
+
+### Account Model
+
+There are 3 kinds of accounts on Solana:
+
+- Data accounts store data
+- Program accounts store executable programs
+- Native accounts that indicate native programs on Solana such as System, Stake, and Vote
+
+Each account has an address (usually a public key) and an owner
+(address of a program account). The full field list an account stores
+is found below.
+
+| Field      | Description                                    |
+|------------|------------------------------------------------|
+| lamports   | The number of lamports owned by this account   |
+| owner      | The program owner of this account              |
+| executable | Whether this account can process instructions  |
+| data       | The raw data byte array stored by this account |
+| rent_epoch | The next epoch that this account will owe rent |
+
+There are a few important ownership rules:
+
+- Only a data account's owner can modify its data and subtract lamports
+- Anyone is allowed to credit lamports to a data account
+- The owner of an account may assign a new owner if the account's data is zeroed out
+
+Program accounts do not store state.
+
+For example, if you have a counter program that lets you increment a counter, you 
+must create two accounts, one account to store the program's code, and one to store 
+the counter.
+
+![](./account_example.jpeg)
+
+To prevent an account from being deleted, you must pay rent.
+
+### Rent
+
+Storing data on accounts costs money to maintain, and it is funded by what is called 
```

Comment:
could be interesting to mention that you can withdraw rent. e.g. this is common for candy machines after the mint has finished

Reply:
Added in 66f1d87914d01c9d854c551a0f81cd00f1919d40